src/bun.js/bindings/headers-handwritten.h

```diff
@@ -239,6 +239,7 @@ extern "C" void BunString__toWTFString(BunString*);
 namespace Bun {
 JSC::JSValue toJS(JSC::JSGlobalObject*, BunString);
 BunString toString(JSC::JSGlobalObject* globalObject, JSC::JSValue value);
+BunString toBunString(const char* bytes, size_t length);
```

Comment:
`toString` instead of `toBunString`?

Reply:
Fixed on [base64URLEncodeToString](https://github.com/oven-sh/bun/pull/5940/commits/2ac4c579f144f43e218158c5e974e588f59c9059)